src/type/printer.js

```diff
@@ -0,0 +1,150 @@
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+var TypeKind = {
```

Comment:
I guess it feels weird to define this struct only to use every value once, compared to just using the strings inline, or reusing an existing struct if it exists.